symphony/app/fbcnms-projects/magmalte/app/views/equipment/EnodebDetailMain.js

```diff
@@ -59,20 +62,36 @@ const useStyles = makeStyles(theme => ({
     textTransform: 'none',
   },
   tabLabel: {
-    padding: '20px 0 20px 0',
+    padding: '16px 0 16px 0',
+    display: 'flex',
+    alignItems: 'center',
   },
   tabIconLabel: {
-    verticalAlign: 'middle',
-    margin: '0 5px 3px 0',
+    marginRight: '8px',
+  },
+  appBarBtn: {
+    color: colors.primary.white,
```

Comment:
we pretty much use these appBarBtn and appBtnSecondary at tons of places. Perhaps can we define this at a single place and import and use that in all the components instead of copy pasting this everywhere?

Reply:
I entirely agree and in a future PR it will be something I want to tackle. The previous comment regarding MuiButton is laying the groundwork, but more needs to be done to make it so that we can replace these buttons.